--- php.ini.apache2	2015-04-10 09:20:05.630023713 -0300
+++ php.ini.new	2015-04-13 09:52:25.121085832 -0300
@@ -476,7 +476,7 @@
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = On
 
 ; The display of errors which occur during PHP's startup sequence are handled
 ; separately from display_errors. PHP's default behavior is to suppress those
@@ -802,7 +802,7 @@
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
-upload_max_filesize = 2M
+upload_max_filesize = 8M
 
 ; Maximum number of files that can be uploaded via a single request
 max_file_uploads = 20
